nyc.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  reporter: ["text", "lcov"],
+  "check-coverage": true,
+  all: true,
+  // https://reflectoring.io/100-percent-test-coverage/
+  branches: 86,
+  lines: 90,
+  functions: 88,
+  statements: 90,
+  // To enable **merged** coverage report all relevant file extensions must be listed.
+  extension: [".js", ".ts", ".vue"],
```

Comment:
if you are using jest for vuejs testing you may not need the .vue
or maybe even the `.js` extensions here.

Reply:
It can stay here for the future maybe. And we run lcov-result-merger.